drivers/staging/vc04_services/bcm2835-audio/bcm2835.c

```diff
@@ -360,10 +360,12 @@ static int snd_bcm2835_alsa_probe(struct platform_device *pdev)
 	    !of_property_read_bool(dev->of_node, "brcm,disable-hdmi"))
 		set_hdmi_enables(dev);
 
-	of_property_read_u32(dev->of_node,
-			     "brcm,disable-headphones",
-			     &disable_headphones);
-	enable_headphones = !disable_headphones;
+	if (enable_headphones) {
```

Comment:
Is this better than the smaller change of `enable_headphones &= !disable_headphones;`?

Reply:
I opted for the proposed implementation to save a DT property read if enable_headphones == 0.